cli/cmd/plugin/ui/web/tanzu-ui/src/shared/components/ConnectionNotification/ConnectionNotification.scss

```diff
@@ -0,0 +1,3 @@
+.hide-me {
```

Comment:
This would be a good shared class to add to index.scss. We have a class in that file named 'hidden' which sets display to none. Maybe we should rename that class to no-display (or something similar) and then declare a class for no-visibility...could tweak naming but that's the general idea

Reply:
Yes that would be better, thanks